spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin/ZipkinSpanListener.java

```diff
@@ -176,9 +164,8 @@ else if (zipkinAnnotation.getValue().equals("release")) {
 	 *
 	 * @return list of Annotations that could be added to Zipkin Span.
 	 */
-	private List<BinaryAnnotation> createZipkinBinaryAnnotations(Span span,
-			Endpoint endpoint) {
-		List<BinaryAnnotation> l = new ArrayList<>();
```

Comment:
removed unnecessary list initializations here and elsewhere, as builders expose addX methods